docs/text/how_to_contribute.rst

```diff
@@ -64,21 +64,32 @@ you have to:
 
 .. code::
 
-    pytest
+    make test-local
 
 
 To test changes across multiple versions of Python, run:
 
 
 .. code::
 
-    tox -r
+    make test-all-local
 
```

Comment:
Similar to above: I am not a big fan of replacing Single-Line commands with a makefile command. This adds another layer of indirection "advanced" users can be annoyed about (they know what tox is, so they want to potentially change options etc) and "new" users copy-paste what is in here anyways, so it does not matter.

Reply:
I do not know for tox, but many tools allow to set default options on the pyproject or setup.cfg or their own config file. This is typically better, because then the tool can do its own "schema" migration. For example, we so this for pytest in the setup.cfg.